Decred Wallet/Features/More/More Features/Debug/Debug.storyboard

```diff
@@ -72,6 +73,26 @@
                                             <segue destination="W5j-lR-v9O" kind="show" id="DcF-yo-afG"/>
                                         </connections>
                                     </tableViewCell>
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="none" accessoryType="disclosureIndicator" indentationWidth="10" reuseIdentifier="connected_peers" textLabel="yh7-MB-dEm" style="IBUITableViewCellStyleDefault" id="TRg-zo-9g9">
+                                        <rect key="frame" x="0.0" y="106" width="375" height="44"/>
+                                        <autoresizingMask key="autoresizingMask"/>
+                                        <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="TRg-zo-9g9" id="L6A-HA-0Qe">
+                                            <rect key="frame" x="0.0" y="0.0" width="348" height="44"/>
+                                            <autoresizingMask key="autoresizingMask"/>
+                                            <subviews>
+                                                <label opaque="NO" multipleTouchEnabled="YES" contentMode="left" insetsLayoutMarginsFromSafeArea="NO" text="View connected peers" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" id="yh7-MB-dEm">
```

Comment:
Text should be localized and text color should be the same as the others.
<img height="200" alt="Screenshot 2021-02-21 at 02 37 44" src="https://user-images.githubusercontent.com/794430/108613039-33004b00-73ee-11eb-952a-d5af714c3b47.png">